behave_test_automation/pah_behave/features/steps/login_steps.py

```diff
@@ -0,0 +1,52 @@
+import parse
+from behave import given, then, register_type
+import features.selectors.login_selectors as login_selector
+import features.selectors.base_selectors as base_selector
+from features.steps.common_actions import short_wait
+
+
+@given('User navigates to pah-fm website')
+def open_main_url(context):
+    context.driver.get('http://localhost:8080/login')
+    current_page = context.driver.current_url
+    assert current_page == 'http://localhost:8080/login'
+
+
+@parse.with_pattern(r".*")
+def parse_nullable_string(text):
+    return text
+
+
+register_type(NullableString=parse_nullable_string)
+
+
+@given('User submits login form with "{login_credential:NullableString}" login and "{'
+       'password_credential:NullableString}" password')
+def submit_form_with_valid_credentials(context, login_credential, password_credential):
+    context.login_credential = login_credential
+    context.password_credential = password_credential
+
+    username_field = context.driver.find_element(*login_selector.username_field)
+    username_field.click()
+    username_field.send_keys(login_credential)
+
+    password_field = context.driver.find_element(*login_selector.password_field)
+    password_field.click()
+    password_field.send_keys(password_credential)
+
+    login_button = context.driver.find_element(*login_selector.login_button_enabled)
+    login_button.click()
+
+
+@then('User is logged in to pah website successfully')
+def successful_login(context):
```

Comment:
AFAIR functions shouls start with verb.

Reply:
Changed to `login_successful` and `login_unsuccessful`